### ISSUE TITLE
Use Meta TX

### ISSUE THREAD

**Issue by vaibhavchellani** [sauravtom/ethereum-url-shortener#1]
Using meta transactions would allow users to create URL without having ether and would be better UX not having to fetch ether before interacting with the dapp ! Great Work ! 
I am happy to collaborate on this if you feel its needed .

**Comment by sauravtom** [comment]
Hi Vaibhav, that's a great idea. I still need to do some reading around Meta transactions. In the meanwhile would you like to create PR for this feature? 🙂

**Comment by vaibhavchellani** [comment]
Sure thing ! We would need to create a centralised server to relay , i guess the current web page is static?

**Comment by sauravtom** [comment]
Yes, the current server is static. But now.sh (hosting provider) also supports node.js and php https://zeit.co/now
